admin_guide/managing_networking.adoc

```diff
@@ -460,10 +460,11 @@ endif::openshift-origin[]
   nodeSelector:
     site: springfield-1 <6>
 ----
-<1> The `pod.network.openshift.io/assign-macvlan annotation` creates a Macvlan
-network interface on the primary network interface, and then moves it into the
-pod's network name space before starting the *egress-router* container. Preserve
-the quotation marks around `"true"`. Omitting them results in errors.
+<1> Creates a Macvlan network interface on the primary network interface, and moves
+ it into the pod's network name space before starting the *egress-router*
+ container. Set to anything other than "true" to specify the network (if using
+ multiple networks). For example, "eth1" Preserve the quotation marks around
+ `"true"`. Omitting them results in errors.
```

Comment:
Hm. I'd put the "preserve the quotation marks" warning before the discussion of alternate interfaces. (Since it only applies to the value `"true"`; you don't need quotation marks around `eth1`, etc, because those aren't YAML keywords.)

And then something like: To create the Macvlan interface on a network interface other than the primary one, set the annotation value to the name of that interface. For example, "eth1".